migrations/versions/3d3fb04d1a98_.py

```diff
@@ -0,0 +1,36 @@
+"""empty message
```

Comment:
Probably a seperate story to "tidy up migrations scripts". I'll put it in as a chore into pivotal.

https://www.pivotaltracker.com/story/show/91376444